packages/discord.js/library/use-reactions.ts

```diff
@@ -0,0 +1,50 @@
+import {
+	Collection,
+	type Message,
+	type MessageReaction,
+	type ReactionCollector,
+} from "discord.js"
+import { useCallback, useEffect, useRef, useState } from "react"
+import { useMessage } from "./use-message"
+
+interface Options {
```

Comment:
It was not meant to be exposed as an interface but I'll also add it as an export in this case 👍

Reply:
It's usually helpful to expose hook arguments for when people make their own wrappers around them and such, otherwise they have to do slightly gross things like `Parameters<typeof useReactions>[0]` 😅